data/src/main/java/com/antosik/benchproject/data/movie/details/entity/relations/MovieAndMovieDetails.kt

```diff
@@ -0,0 +1,15 @@
+package com.antosik.benchproject.data.movie.details.entity.relations
+
+import androidx.room.Embedded
+import androidx.room.Relation
+import com.antosik.benchproject.data.movie.details.entity.MovieDetailsEntity
+import com.antosik.benchproject.data.popular.movies.entity.MovieEntity
+
+internal data class MovieAndMovieDetails(
```

Comment:
Maybe `MovieWithDetails`?

Reply:
It's a relation `one-to-one` I watched a movie tutorial and the convention looks like the `TableEntity1AndTableEntity2` convention for `one-to-many` looks like the `TableEntity1WithTableEntity2` also I saw this convention on this page https://developer.android.com/training/data-storage/room/relationships#one-to-one what do you think about that?

Reply:
In the link I send above, you can find two ways with naming relation class. 😄

Reply:
Okay, so in this case it would be good to follow that convention 😄